fairytale.md

```diff
@@ -1,4 +1,31 @@
 # MNXB11 fairytale
 
 Once upon a time,
-there was a **beautiful** _princess_
+there was a **ruthless** _hogre_
```

Comment:
Typo: Should be "ogre" not "hogre".

Reply:
Oops! sure! fixing

Reply:
Fixed in 21f6d5a55b853c63c140505e1c16bf3d68552593